pallet/erc20-peg/src/lib.rs

```diff
@@ -517,7 +526,7 @@ impl<T: Config> Pallet<T> {
 
 		// Process transfer or withdrawal of payment asset
 		let _imbalance = Self::burn_or_transfer(asset_id, &origin, amount)?;
-		Self::process_withdrawal(message, asset_id)
+		Self::process_withdrawal(origin.clone(), message, asset_id)
```

Comment:
nit: dont need clone, can take ownership
```suggestion
		Self::process_withdrawal(origin, message, asset_id)
```

Reply:
https://github.com/futureversecom/trn-seed/pull/887/commits/9fae984a1b4495afa30219d4e3fbb6c5fc4f3d00